app/lib/.server/llm/providers/ollama.ts

```diff
@@ -0,0 +1,15 @@
+import { createOllama } from 'ollama-ai-provider';
+import type { ModelFactory } from './modelFactory';
+
+export function getOllamaModel(apiKey: string, modelName: string = 'llama3.2:latest') {
+  const model = createOllama({
+    baseURL: 'http://172.21.208.1:11434',
```

Comment:
I think this should be in an environmental variable

Reply:
I abandoned this PR in favor of [this fork](https://github.com/coleam00/bolt.new-any-llm)
due to inactivity of developers here.

I recommend you to use this fork instead.